quixstreams/sources/community/file/origins/local.py

```diff
@@ -0,0 +1,33 @@
+from io import BytesIO
+from pathlib import Path
+from typing import Generator
+
+from .base import Origin
+
+__all__ = ("LocalOrigin",)
+
+
+class LocalOrigin(Origin):
+    def __init__(
+        self,
+    ):
+        self._client = None
```

Comment:
No need for client here anymore, right?

Reply:
Correct, removed all the extra stuff, forgot to remove it =)